vue/src/ContainerSelector/ContainerSelector.less

```diff
@@ -23,4 +24,10 @@
             }
         }
     }
-}
\ No newline at end of file
+}
+
+.siteSelector {
```

Comment:
@ulcuber This will change the core too can you
wrap it under `tagmanager` class, something like this
```
.tagmanager {
  .siteSelector {
    a.title {
      max-width: ~'calc(58vw - 250px - 1em)';
    }
  }
}
```